### ISSUE TITLE
Formatting does not work

### ISSUE THREAD

**Issue by BorzdeG** [shyiko/ktlint#296]
ktlint version: 0.29.0

source file `Test.kt`:
```kotlin
class Test {
val t = "test"

fun testF() {
 println("test")
}
}
```

`.editorconfig`:
```
root = true

[*]
indent_style = space
indent_size = 2
end_of_line = lf
charset = utf-8
trim_trailing_whitespace = true
insert_final_newline = true
```

command run: `./ktlint -F -y --debug Test.kt`:
```
[DEBUG] Discovered ruleset "standard"
[DEBUG] Discovered reporter "plain"
[DEBUG] Discovered reporter "json"
[DEBUG] Discovered reporter "checkstyle"
[DEBUG] Initializing "plain" reporter with {verbose=false, color=false}
[DEBUG] Checking CUT_PROJECT_PATH/Test.kt
[DEBUG] Discovered .editorconfig (CUT_PROJECT_PATH) {charset=utf-8, end_of_line=lf, indent_size=2, indent_style=space, insert_final_newline=true, trim_trailing_whitespace=true}
[DEBUG] 1512ms / 1 file(s) / 0 error(s)
```

actual result: no changes
expected result:
```kotlin
class Test {
  val t = "test"

  fun testF() {
    println("test")
  }
}
```

**Comment by BorzdeG** [comment]
It seems I misunderstood the work of ktlint - the check also does not swear at the absence of indents ...

`./ktlint --debug Test.kt` does not produce errors.

**Comment by shyiko** [comment]
This is something @iboss-ptk is working on. Track progress in https://github.com/shyiko/ktlint/pull/294.

**Comment by shyiko** [comment]
Moved to #338.
